__tests__/app.test.js

```diff
@@ -23,32 +42,51 @@ describe("/api/topics", () => {
   });
 });
 
-describe("Generic handler", () => {
-  test("GET: 404 sends an appropriate status and error message.", () => {
+describe("/api/articles", () => {
+  test("GET: 200 sends all articles in the correct format.", () => {
     return request(app)
-      .get("/api/top")
-      .expect(404)
+      .get("/api/articles")
+      .expect(200)
       .then((response) => {
-        expect(response.body.msg).toBe("Path not found.");
+        const { articles } = response.body;
+        expect(articles.length).toBe(13);
+        expect(Array.isArray(articles)).toBe(true);
+        articles.forEach((article) => {
+          expect(typeof article).toBe("object");
+        });
       });
   });
-});
-
-describe("/api", () => {
-  test("GET: 200 sends an object with all available endpoints.", () => {
+  test("Should return articles without a body, and with a commment_count.", () => {
     return request(app)
-      .get("/api")
+      .get("/api/articles")
       .expect(200)
       .then((response) => {
-        expect(typeof response.body).toBe("object");
+        const { articles } = response.body;
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("comment_count");
+        });
       });
   });
-  test("Should check that all available APIs are included.", () => {
+  test("Should return articles with correct comment_count.", () => {
     return request(app)
-      .get("/api")
+      .get("/api/articles")
       .expect(200)
       .then((response) => {
-        expect(response.body.endpoints).toEqual(endpoints);
+        const { articles } = response.body;
+        expect(articles[0].comment_count).toBe(2);
+      });
+  });
+  test("Should return sorted articles in desc order by date.", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        const latest = articles[0].created_at;
+        const earliest = articles[12].created_at;
+
+        expect(latest > earliest).toBe(true);
```

Comment:
Check out the npm package `jest sorted` as it's tailor made for asserting on the sort order of arrays and would be more effective